lib/dao.js

```diff
@@ -46,109 +38,156 @@ export class Organization {
       options.founders.map(x => x.address),
       options.founders.map(x => x.tokens),
       options.founders.map(x => x.reputation),
-      NULL_ADDRESS
+      Utils.NULL_ADDRESS
     );
     // get the address of the avatar from the logs
-    const avatarAddress = getValueFromLogs(tx, "_avatar", "NewOrg");
-    const org = new Organization();
+    const avatarAddress = Utils.getValueFromLogs(tx, "_avatar", "NewOrg");
```

Comment:
get the log values directly from the transaction return type

Reply:
This is not coming from a wrapper (yet), so the suggestion is not yet possible.